homeassistant/requirements.py

```diff
@@ -3,22 +3,29 @@
 import logging
 import os
 from pathlib import Path
-from typing import Any, Dict, Iterable, List, Optional, Set
+from typing import Any, Dict, Iterable, List, Optional, Set, Union, cast
 
 from homeassistant.core import HomeAssistant
 from homeassistant.exceptions import HomeAssistantError
-from homeassistant.loader import Integration, async_get_integration
+from homeassistant.loader import (
+    Integration,
+    IntegrationNotFound,
+    _async_mount_config_dir,
```

Comment:
Importing this unused name is a lint error.

Reply:
My bad. Fixed it.